.circleci/config.yml

```diff
@@ -57,6 +57,9 @@ jobs:
     steps:
       - checkout
       - *restore_cache
+      - run:
+          name: Approve license for build tools
+          command: (echo y; echo y; echo y; echo y; echo y; echo y) | $ANDROID_HOME/tools/bin/sdkmanager --licenses
```

Comment:
You shouldn't need to accept licences here since this is using the google maven. 

What was requesting you to accept licences?

Reply:
Whats with the port over to circleci?

Reply:
Added CircleCi to run checkstyles & PMD originally, to notify contributors if they've violated any rules before merging. Obviously, we'll need to update to use some Kotlin equivalent now.